packages/flutter/lib/src/material/search.dart

```diff
@@ -184,6 +190,13 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.actions], the intended use for the return value of this method.
   List<Widget> buildActions(BuildContext context);
 
+  /// Widget to display across the bottom of the [AppBar].
+  ///
+  /// See also:
+  ///
+  ///  * [AppBar.bottom], the intended use for the return value of this method.
+  PreferredSizeWidget buildBottom(BuildContext context);
```

Comment:
This would be a breaking change, since existing apps will not include an override for this method. Should be OK to just return null by default (and document as much).

Reply:
Updated documentation.